lib/std/Build/Step/Options.zig

```diff
@@ -151,12 +151,27 @@ fn printType(options: *Options, out: anytype, comptime T: type, value: T, indent
     }
 
     switch (@typeInfo(T)) {
-        .array => {
+        .array => |info| {
+            try printUserDefinedType(options, out, info.child, 0);
+
+            const type_name = @typeName(T);
+
             if (name) |some| {
-                try out.print("pub const {}: {s} = ", .{ std.zig.fmtId(some), @typeName(T) });
+                // If the type name doesn't contains a '.' the type is from zig builtins.
+                if (std.mem.containsAtLeast(u8, type_name, 1, ".")) {
```

Comment:
sorry, I won't merge any patch that depends on searching type names for substrings.